docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+\chapter{Developer Guide}
```

Comment:
Overall comment:
Maybe good if we had a look together in what order we would expect the developer guide to go.
e.g.
# setting up enviroment.
# running a test
# running coverage
# debugging a test
# information regarding dependencies (poetry & toml)
# adding/updating dependencies
# Nuitka build?

Reply:
see DFAST-132